kuma/wiki/models.py

```diff
@@ -1492,6 +1493,10 @@ def zone_stack(self):
     def get_full_url(self):
         return absolutify(self.get_absolute_url())
 
+    @property
+    def is_experiment(self):
+        return self.slug.startswith(EXPERIMENT_TITLE_PREFIX)
```

Comment:
Testing locally, I cloned an article (`docs/Web/JavaScript/Reference/Functions/Arrow_functions`), which resulted in a new article with the following URL: `docs/Web/JavaScript/Reference/Functions/Experiment:_Arrow_functions`

As the slug doesn't start with `Experiment:`, this article is not flagged as such. Did I do something wrong/unanticipated, or do we need to see if the slug contains `Experiment:` (as opposed to `startswith`)?